src/TLuaInterpreter.cpp

```diff
@@ -842,25 +842,28 @@ int TLuaInterpreter::selectCaptureGroup(lua_State* L)
         lua_pushnumber(L, -1);
         return 1;
     }
-    luaNumOfMatch--; //we want capture groups to start with 1 instead of 0
-    if (luaNumOfMatch < static_cast<int>(host.getLuaInterpreter()->mCaptureGroupList.size())) {
+    // We want capture groups to start with 1 instead of 0 so predecrement
+    // luaNumOfMatch :
```

Comment:
Might as well delete this line, else it'll be confusing as to why it was left in there later.

Reply:
The comment is still relevant - but it now applies to the following line (besides _pre_-{incre|dec}ments are more efficient than their _post_-variants)...

Reply:
I meant the `// luaNumOfMatch;` bit